src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -1,13 +1,28 @@
+import { getUrlPath } from '#lib/utilities/urlParser';
+import isLive from '#lib/utilities/envHelper';
+import hasLiveOverride from '#lib/utilities/rendererEnvHelper';
+
+const AV_ROUTE = 'ws/av-embeds';
+
+const getBaseUrl = requestUrl => {
+  if (hasLiveOverride(requestUrl) || isLive()) {
```

Comment:
`requestURL` here doesn't seem to be the URL of the page

eg, if I make a request to `http://localhost:7080/pidgin/23248703?renderer_env=live`
`requestUrl` in this function is `pidgin/23248703/p01kx42v/pcm`

I don't think `hasLiveOverride(requestUrl)` will ever be true, so we never switch to the live endpoint

Reply:
Good spot - the pageURL is now passed to embedUrl as a param, so we can determine what the override is.